.changes/unreleased/Fixed-20230715-131052.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Fixed
+body: Bump engine alpine version to fix service DNS resolution
+time: 2023-07-15T13:10:52.157510517-03:00
+custom:
+  Author: marcosnils
+  PR: "5470"
```

Comment:
I don't think that we want this in user-facing release notes, since it doesn't impact them. WDYT?

Reply:
This PR bumps the base image for our engine image, so it would impact any users that have `search .` in their `/etc/resolv.conf`. That seems to be somewhat a common Linux configuration, so I think it's worth including.